notebooks/writing_a_trace_transform_cpu_offloading.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        Parameters:
</pre><p>IMHO this repo usually follows <a href="https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings" target="_blank">https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings</a> in dostring. So how about Args: instead for the consistency with the existing docstrings?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/966/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>